src/TurboLinks.Net/TurboLinks.cs

```diff
@@ -24,13 +24,13 @@ public TurboLinks(RequestDelegate next)
             var request = context.Request;
             var response = context.Response;
 
-            if(!string.IsNullOrWhiteSpace(request.Headers["X-XHR-Referer"]))
+            if (!string.IsNullOrWhiteSpace(request.Headers["X-XHR-Referer"]))
```

Comment:
Please remove the space, this is changing the style rules of the repo for no reason.